ui/components/molecules/M008-Popper/schema.js

```diff
@@ -0,0 +1,37 @@
+export default {
+  $schema: 'http://json-schema.org/schema#',
+  type: 'object',
+  properties: {
```

Comment:
Im not sure we need all of these. It feels like you are just taking all the options out of the docs and putting them in the schema. We actually want the smallest number of things here. Only the thing we are very sure we are going to use. eg no modifiers, transition or anchorE1. ALl of these should be hard coded to provide consistency across the components use and reduce the amount of work the future uses of the component need to do.